bucketPolicy/utils.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Exit upon encountering an error
+set -euo pipefail
+
+# Set some enviroment configurations
+BOLD='\e[1m'
+BLUE='\e[34m'
+RED='\e[31m'
```

Comment:
`Colour` variables such as these that are unused should be removed from the script